doc/reference_measure.xml

```diff
@@ -820,6 +820,64 @@ SELECT degrees(ST_Azimuth(ST_Point(25, 45), ST_Point(75, 100))) AS degA_B,
 		</refsection>
 
   </refentry>
+  
+  	<refentry id="ST_Angle">
+		<refnamediv>
+		  <refname>ST_Angle</refname>
+
+		  <refpurpose>Returns the angle in radian measured clokwise between 3 points, or the angle measured clockwise between 2 vectors.</refpurpose>
+		</refnamediv>
+		<refsynopsisdiv>
+		  <funcsynopsis>
+			<funcprototype>
+			  <funcdef>float <function>ST_Angle</function></funcdef>
+			  <paramdef><type>geometry </type><parameter>point1</parameter></paramdef>
+			  <paramdef><type>geometry </type><parameter>point2</parameter></paramdef>
+			  <paramdef><type>geometry </type><parameter>point3</parameter></paramdef>
+			  <paramdef><type>geometry </type><parameter>point4</parameter></paramdef>
+			</funcprototype> 
+		  </funcsynopsis>
+		</refsynopsisdiv>
+		<refsection>
+			<title>Description</title>
+
+			<para> For 3 points, computes the angle measured clockwise of P1P2P3.
+			For 4 points,compute the angle measured clockwise of P1P2,P3P4.
+			Results are always positive, between 0 and 2*Pi radians.
+			Uses ST_Azimuth.
```

Comment:
"Uses ST_Azimuth" isn't really true, it's using liblwgeom internal functions (on which ST_Azimuth is also based).